src/sbtc-releases/sbtc-dev.md

```diff
@@ -1,3 +1,106 @@
+6. Pool
+7. Hand-off
+8. Bitcoin library
+
+### Controller
+
+The controller is a single Clarity smart contract that manages which other contracts are part of the protocol. The other contracts use the controller for access control. The controller also exposes a pathway to upgrade the protocol by changing the access list.
+
+### Registry
+
+The registry contract functions as the main storage space for the protocol. The other components use the registry to store protocol state. It is intended to allow for more seamless upgrades by keeping the registry the same across future soft upgrades.
+
+[List the things the registry stores and why, also how those things are updated. Pay special attention to the get-and-update function.]
+
+### Token
+
+The sBTC token is implemented as a SIP-010 fungible token. It uses the controller for access control so that the sBTC Protocol contracts may mint and burn sBTC tokens.
+
+### Deposit
+
+The deposit contracts verify Bitcoin deposit transactions. They trigger an sBTC mint to the recipient found in the Bitcoin transaction. The transaction format is described in the [section on Bitcoin transactions](../sbtc-operations/bitcoin-transactions.md).
+
+[Explain how deposits are processed. Link back to BTC transaction structure and minimum confirmations required.]
+
+### Withdraw
+
+The withdrawal contracts process withdrawal requests and verify Bitcoin withdrawal transactions.
+
+Withdrawal requests start on Stacks. However, an sBTC holder that does not own any STX can still sign a Stacks transaction and pay transaction fees in sBTC. The protocol exposes a mechanism to submit the withdrawals by means of a sponsored Stacks transaction. The sponsor will then receive an amount of sBTC in return for sponsoring the transaction. A sponsor can decide to sponsor the transaction if they find the fee acceptable.
+
+[Explain steps and how redemptions are processed. Link back to BTC transaction structure and minimum confirmations required. Explain more what the process looks like.]
+
+### Pool
+
+The sBTC Stacking pool proper that allows Stackers to participate in the sBTC PRotocol. Stackers will participate in the distributed key generation scheme and individually vote on the next Bitcoin wallet address to use. They vote by locking their STX for the next cycle.
+
+[Is it deserving of a sub page? Will be a lot of information here on how to use the Stacking pool and voting.]
+
+### Hand-off
+
+The contract that verifies that the hand-off to the next Bitcoin wallet was done properly.
+
+[Mostly automated, the Signer binary (or who?) will submit hand-off proof Stacks transaction.]
+
+### Bitcoin library
+
+The helper library that makes it easier to decode and interpret Bitcoin transaction in Clarity. Any component that receives Bitcoin transactions uses this library. It is an adapted version of an earlier open source library [link and attribution here].
+
+## Contracts
+
+[Does this section go here? It will be a long document.]
```

Comment:
I'd recommend starting here and break it out to a separate page (or pages) depending on how long it becomes and what it ends up containing.